telegram/ext/_commandhandler.py

```diff
@@ -71,7 +71,7 @@ class CommandHandler(Handler[Update, CCT]):
         ValueError: when command is too long or has illegal chars.
 
     Attributes:
-        command (:obj:`str` | Tuple[:obj:`str`] | List[:obj:`str`]):
+        command (:obj:`str` | Tuple[:obj:`str`] | List[:obj:`str`] | Set[:obj:`str`]):
```

Comment:
In the current implementation, `CommandHandler.command` is actually always a `List` …

Reply:
> Hi. thanks for the PR! IISC so far you have only allowed for the `PrefixHandler.prefix/command` setters to accept sets. I think I should clarify a bit what the intention of #2885 was:
> 
> * The main idea of the issue was to rename `CommandHandler.command` to `CommandHandler.commands` (plural) since that attribute is currently actually always a list. Extending this to `PrefixHandlers.prefix` is surely reasonable.
> * Since I noticed that we only ever need `x in self.command`, my idea was to make `CommandHandler.commands` a set instead of a list. Accepting sets as input in addition is surely a logical extension. So
> 
> Additionally, while reviewing your PR, I noticed that unlike `PrefixHandler.command`, `CommandHandler.command` doesn't have a setter. I.e. if you do `command_handler.command = 'NewCommand'`, it won't be converted to `[newcommand]`. I think we should fix that. So what I would ask you to do is:
> 
> * Make sure that sets are accepted as input for `Prefix/CommandHandler.command` and `PrefixHandler.prefix` - this includes type hints & documentation
> * rename `CommandHandler.command` to `CommandHandler._commands` and add getter + setter proprties `CommandHandler.commands`. The getter should provide access to the set `CH._commands` and the setter should accept a single string or tuple/list/set and convert it as it is converted on `__init__`
> * Apply analogous changes to `PrefixHandler`

Thanks for the review, I'm working on it. should I change the name in arguments from command to commands too?